tests/end-to-end/backup/pkg/tests/eventmesh/event.go

```diff
@@ -17,7 +21,51 @@ import (
 	duckv1 "knative.dev/pkg/apis/duck/v1"
 )
 
-func SendEvent(target, payload, eventType, eventTypeVersion string) error {
+type LegacyEvent struct {
+	EventType        string    `json:"event-type"`
+	EventTypeVersion string    `json:"event-type-version"`
+	EventID          string    `json:"event-id"`
+	EventTime        time.Time `json:"event-time"`
+	Data             string    `json:"data"`
+}
+
+func SendLegacyEvent(target, payload, eventType, eventTypeVersion string) error {
+	event := LegacyEvent{
```

Comment:
(nit)
```suggestion
	event := EventBusV1{
```

Reply:
i chose legacy explicitly, as it states that this one is the old way to send stuff